sdk/ml/azure-ai-ml/azure/ai/ml/operations/_job_operations.py

```diff
@@ -769,7 +769,7 @@ def _get_named_output_uri(
             try:
                 job = self.get(job_name)
                 artifact_store_uri = job.outputs[DEFAULT_ARTIFACT_STORE_OUTPUT_NAME]
-                if artifact_store_uri and artifact_store_uri.path:
+                if artifact_store_uri is not None and artifact_store_uri.path:
```

Comment:
As download test is only running in live mode, so it's not early detect such error?

Reply:
No, existing download test will not cover this logic. Need some investigation on how to cover this from test.